src/main/resources/static/api.yaml

```diff
@@ -43,7 +45,59 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/PatientDiscoveryResponse'
+                $ref: '#/components/schemas/PatientDiscoveryRepresentation'
+
+  # consent-service
+  /consent-requests:
+    get:
+      tags:
+        - consent
+      summary: List consent requests
+      description: Return list of consent requests which are matching filter criteria.
+      operationId: listConsentRequests
+      parameters:
+        - $ref: '#/components/parameters/authorization'
+        - $ref: '#/components/parameters/limit'
+        - $ref: '#/components/parameters/offset'
+        - $ref: '#/components/parameters/filters'
+      responses:
+        '200':
+          description: List of consent requests.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ConsentRequestsRepresentation'
+  /consent-requests/{request-id}/approve:
+    post:
+      tags:
+        - consent
+      summary: Approve Request
+      description: User approves the consent request with preferred data access parameters.
+      parameters:
+        - $ref: '#/components/parameters/authorization'
+        - name: request-id
+          in: path
+          required: true
+          schema:
+            type: string
+            format: uuid
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ConsentApproveRequest'
```

Comment:
this can be an array of ConsentApproveRequest

Reply:
@angshu  How will the response look like?